src/app/(user)/search/_components/search-video-card.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+import React, { useEffect, useState } from "react";
+import { DashboardCategory, Profile, videoData } from "@/lib/searchTypes";
```

Comment:
why importing this when it is not being used except videoData, change this across